internal/capability/openc2/openc2.go

```diff
@@ -16,6 +17,7 @@ type OpenC2Capability struct {
 type Empty struct{}
 
 const (
+	maxResultVariables       = 1
```

Comment:
How/where is this used? I don't see it in the other capabilities or in the rest of the code of this capability but I haven't looked in depth

Reply:
It is not used it remained of my first implementation so I will remove it good catch